.github/workflows/zjit-ubuntu.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
           - test_task: 'zjit-test-all'
             configure: '--enable-zjit=dev'
-            testopts: '--seed=11831'
+            testopts: '--seed=18140'
```

Comment:
With different tests running, ubuntu now needs a different seed to pass.
The old seed was causing [this crash](https://github.com/ruby/ruby/actions/runs/16329885791/job/46129574671).